cnd/src/http_api.rs

```diff
@@ -143,41 +143,32 @@ impl Amount {
 
 #[derive(Serialize)]
 struct State {
-    #[serde(serialize_with = "percent_string")]
-    open: u8,
-    #[serde(serialize_with = "percent_string")]
-    closed: u8,
-    #[serde(serialize_with = "percent_string")]
-    settling: u8,
-    #[serde(serialize_with = "percent_string")]
-    failed: u8,
-    #[serde(serialize_with = "percent_string")]
-    cancelled: u8,
+    open: Amount,
+    closed: Amount,
+    settling: Amount,
+    failed: Amount,
+    cancelled: Amount,
 }
 
 impl State {
     #[allow(clippy::cast_sign_loss, clippy::cast_possible_truncation)] // we only store positive values in the DB ranging from 0 - 100
```

Comment:
📣 
```suggestion
```

Reply:
This check concerned the validity of the percentages and is not relevant any more.

Reply:
I guessed :)